### PR TITLE
Readme update: change dps to customer language

### PR DESCRIPTION
**Pull request by jenniferlynparsons**
## Description

This updates the Readme for the project to change the obsolete DPS setup information to the current Customer user setup. Steps are included to cover both initial project setup and database reset use cases for the information.

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](tbd) for this change
* [this article](tbd) explains more about the approach used.

## Screenshots

If this PR makes visible UI changes, an image of the finished UI can help reviewers and casual
observers understand the context of the changes. A before image is optional and
can be included at the submitter's discretion.

Consider using an animated image to show an entire workflow instead of using multiple images. You may want to use GIPHY CAPTURE for this! 📸

_Please frame screenshots to show enough useful context but also highlight the affected regions._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -315,10 +315,19 @@ Dependencies are managed by yarn. To add a new dependency, use `yarn add`
...
+
+This step is only necessary if you are running migrations after a reset and do not see any sample users in the database.
+
+1. Run `make db_dev_e2e_populate`. This will load the `devseed` data into your local database.
+
+2. Log into [MilMove Devlocal Auth](http://milmovelocal:3000/devlocal-auth/login) and a list of seeded users should be visible.
+
+If you would like to create a new single test user:
 
 1. Ensure that you have a login.gov test account
```
Comment (kimallen) [inline README.md:328]: I think this step could actually be removed.  If we are using the local login, we don't need a login.gov test account.

**Review by kimallen — COMMENTED** [review]
Thanks so much for making these modifications! It will definitely help future onboarders be less confused.

I can't seem to put a comment in unchanged code, but I'm also seeing that the table of contents could use copy change from DPS to Customer (line 49)

File: README.md [hunk 2]
```diff
@@ -315,10 +315,19 @@ Dependencies are managed by yarn. To add a new dependency, use `yarn add`
 
 1. `make admin_client_run`
 
-### Setup: DPS user
+### Setup: Customer user
+
+This step is only necessary if you are running migrations after a reset and do not see any sample users in the database.
```
Comment (duncan-truss) [inline README.md:320]: Running `make db_dev_e2e_populate` will reset your database and apply migrations before loading all of the devseed data so maybe this is more confusing than helpful.  I think it's more an optional step if you want to log in as an existing user and your database is empty.  Then you still have the options of creating a new devlocal user or creating a new user through login.gov.

File: README.md (hunk 1 above)
Comment (duncan-truss) [inline README.md:328]: Creating a new (not devlocal) MilMove user means creating a new login.gov account every time so nothing would need to exist previously

File: README.md [hunk 3]
```diff
@@ -315,10 +315,19 @@ Dependencies are managed by yarn. To add a new dependency, use `yarn add`
...
+
+1. Run `make db_dev_e2e_populate`. This will load the `devseed` data into your local database.
+
+2. Log into [MilMove Devlocal Auth](http://milmovelocal:3000/devlocal-auth/login) and a list of seeded users should be visible.
```
Comment (duncan-truss) [inline README.md:324]: The users should all be listed without logging in.  By selecting one of them from the list we're logging in locally without the need for a password.

File: README.md [hunk 4]
```diff
@@ -315,10 +315,19 @@ Dependencies are managed by yarn. To add a new dependency, use `yarn add`
...
+
+If you would like to create a new single test user:
 
 1. Ensure that you have a login.gov test account
-2. Log into [MilMove Devlocal Auth](http://milmovelocal:3000/devlocal-auth/login) and create a new DPS user from the interface.
+2. Log into [MilMove Devlocal Auth](http://milmovelocal:3000/devlocal-auth/login) and click "Create a New milmove User" from the interface.
+3. Create the test user with sample data. Look in either `pkg/testdatagen/scenario/e2ebasic.go` or `devseed.go` to see some different examples of test users.
```
Comment (gidjin) [inline README.md:330]: You shouldn't modify `e2ebasic.go` unless you are adding data or users for the `cypress` tests. For local developer use make your changes in `devseed.go`

**Pull request closed without merging**